spec/requests/api/v1/merchants_request_spec.rb

```diff
@@ -118,12 +118,12 @@
   end
 
   describe "Get all of a merchant's items by the merchant's id" do
-    xit "renders a JSON representation of all records of the requested resource" do
+    it "renders a JSON representation of all records of the requested resource" do
```

Comment:
Great job working through this. This is well written and does a lot of heavy logic lifting.